buildout.d/versions.cfg

```diff
@@ -2,6 +2,7 @@
 setuptools = 8.3
 zc.buildout = 2.2.5
 zc.recipe.egg = 2.0.1
+plone.app.widgets = 1.8.0
```

Comment:
Ok. We can leave it like this for now. Though, there must be another package that fetches plone.app.widgets (which is wrong).